### PR TITLE
MIlestone One:  Create project and deploy to Heroku

### PR DESCRIPTION
**Pull request by nganifaith**
Project setup using.
project created using create-react-app --use-npm command
deployment to Heroku

### CODE REVIEW

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi Faith, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. :pray: 

## Invalid Code Review Request :red_circle: 

You have submitted a project without a custom README. 
Friendly reminder: You should make sure that your repo looks professional before review.
Please take a step back and read a [lesson here.](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/reportWebVitals.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
-const reportWebVitals = onPerfEntry => {
+const reportWebVitals = (onPerfEntry) => {
   if (onPerfEntry && onPerfEntry instanceof Function) {
-    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
       getCLS(onPerfEntry);
       getFID(onPerfEntry);
       getFCP(onPerfEntry);
```
Comment (george-shammar) [inline src/reportWebVitals.js:8]: - [ ] Let's remove unnecessary boiler plate files created by create-react-app. As per the project requirement, unnecessary files or assets, such as the React svg Logo should be deleted.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi Faith, 

Good job so far! :clap:
 
You're on your way to creating a cool application. :+1: 

There are some issues that you still need to work on to go to the next milestone but you are almost there!

## Required Changes ♻️
_Check the comments under the review section as well._

- [x] Unnecessary files or assets, such as the React svg Logo, should be deleted. `setupTests.js`, `App.test.js`, `reportWebVitals.js` and all image files all belong to this category.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by nganifaith** [comment]
> Hi Faith,
> 
> Good job so far! 👏
> 
> You're on your way to creating a cool application. 👍
> 
> There are some issues that you still need to work on to go to the next milestone but you are almost there!
> 
> ## Required Changes ♻️
> _Check the comments under the review section as well._
> 
> * [x]  Unnecessary files or assets, such as the React svg Logo, should be deleted. `setupTests.js`, `App.test.js`, `reportWebVitals.js` and all image files all belong to this category.
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

@george-shammar  thank you for your review. Some of these files mentioned will be necessary for my project. reportWebVital.js will be for performance in my application. 

- setupTests.js will be used in testing my application. 
- Finally, I am using the logo in my index.html

**Review by rindrajosia — APPROVED** [review]
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**